samples/assistant/csharp-ooproc/AssistantApis.cs

```diff
@@ -13,6 +13,9 @@ namespace AssistantSample;
 /// </summary>
 static class AssistantApis
 {
+    const string CHAT_STORAGE_CONNECTION_SETTING = "AzureWebJobsStorage";
+    const string COLLECTION_NAME = "SampleChatState";
```

Comment:
nit: The C# convention for constants (unlike Java or C/C++) is to simply use PascalCase.